File: fundamentals/array_access.exs
defmodule ArrayAccess do
  def execute(index) do
    arr = [1, 2, 3, 4, 5]

    IO.puts("Array: #{inspect(arr)}")
    IO.puts("Accessing index \e[33m#{index}\e[0m")

    value = Enum.at(arr, index)

    IO.puts("Value: \e[32m#{value}\e[0m\n")
    value
  end
end

File: fundamentals/ArrayAccess.java
package fundamentals;

public class ArrayAccess {
    public static int execute(int index) {
        int[] arr = { 1, 2, 3, 4, 5 };
        System.out.print("Array: [");
        for (int i = 0; i < arr.length; i++) {
            System.out.print(arr[i]);
            if (i < arr.length - 1)
                System.out.print(", ");
        }
        System.out.println("]");
        System.out.println("Accessing index \u001B[33m" + index + "\u001B[0m");
        System.out.println("Value: \u001B[32m" + arr[index] + "\u001B[0m\n");

        return arr[index];
    }

    public static void ArrayAccess(String[] args) {
        execute(2);
    }
}
